libraries/AP_BattMonitor/AP_BattMonitor_ADS1115.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <AP_ADC/AP_ADC.h>                 // ArduPilot Mega Analog to Digital Converter Library
+#include "AP_BattMonitor.h"
+#include "AP_BattMonitor_Backend.h"
+#include <cstdint>
+#include <vector>
+#include <tuple>
+
+
+const int AP_BATTMONITOR_SMBUS = 0;
```

Comment:
Does this need to be a 'const int'? Any reason it couldn't be a #define in similar fashion to my previous comment about the #define in the linux.h file? Also, since this is just defining the i2c bus in this new driver, give it a more descriptive name (SMBUS is a different driver, right?).

Reply:
I was trying to keep the Battery Monitor encapsulated, so it would only affect the AP_BattMonitor_ADS1115.h, but you are correct, we can add it to the linux.h file with little consequence.